docs/src/apalache/index.md

```diff
@@ -1,20 +1,27 @@
-# Apalache Manual
+# Getting Started
 
-**Authors: Igor Konnov, Jure Kukovec, Andrey Kuprianov, Shon Feder**
+Apalache is a symbolic model checker for the [specification language][]
+[TLA+][]. As such, it is a recent alternative to the explicit-state model
+checker [TLC][].
 
-**Contact: {igor,jure,andrey,shon} at informal.systems**
+## Apalache vs. TLC
 
-Apalache is a symbolic model checker for
-[TLA+](https://lamport.azurewebsites.net/tla/tla.html). (*Still looking for a
-better tool name.*) Our model checker is a recent alternative to
-[TLC](https://lamport.azurewebsites.net/tla/tools.html?unhideBut=hide-tlc&unhideDiv=tlc).
 Whereas TLC enumerates the states produced by the behaviors of a TLA+
 specification, Apalache translates the verification problem to a set of logical
-constraints. These constraints are solved by an [SMT
-solver](https://en.wikipedia.org/wiki/Satisfiability_modulo_theories), for
-instance, by [Microsoft's Z3](https://github.com/Z3Prover/z3). That is, Apalache
-operates on formulas (i.e., _symbolically_), not by enumerating states one by one
-(i.e., _state enumeration_).
+constraints. These constraints are solved by an [SMT solver][SMT], for example,
+by [Microsoft Z3][]. That is, Apalache operates on formulas (i.e.,
+_symbolically_), not by enumerating states one by one (i.e., _state
+enumeration_).
+
+### Shall I use Apalache or TLC?
+
+We recommend starting with TLC. It is mature, well-documented, and
+well-integrated into TLA+ Toolbox. Once you have debugged your TLA+
+specification, and TLC is still producing too many reachable states, switch to
+Apalache. We are using this approach at [Informal Systems][].
```

Comment:
Shall we remove this paragraph too? As we have found, people would write more efficient specs for Apalache, if they start with Apalache. So the old workflow is less obvious to me now.

Reply:
Maybe we should include something like the snippet we wrote for the TLA+ website?

Something like
> Depending on the specification you wrote, either TLC or Apalache may be more efficient in checking it. While TLC is a mature tool, Apalache is still experimental, so be prepared to use the command-line and to help us discover bugs.

Reply:
Yes, this looks good.